tools/mg_toolkit/.shed.yml

```
@@ -0,0 +1,9 @@
+name: mg-toolkit
```

```suggestion
name: mg_toolkit
```

dashes are not allowed, see the lining failures